andela/ah-alpha-frontend
# #162948864 Create an empty react app

#### What does this PR do?
- Setup an empty react project
- Setup Airbnb as our default lint guide
#### Description of Task to be completed?
Setup the basic structure of the frontend 
-Create new app by running`npx create-react-app <name_of_app>` then `cd <name_of_app>`
#### How should this be manually tested?
- Clone the repository `git clone`
- git checkout `ch-create-react-app-162948864`

- To check for linting errors run `npm run lint`

-  To run the app run `npm start` 

#### What are the relevant pivotal tracker stories?
[#162948864](https://www.pivotaltracker.com/story/show/162948864)
